materialized_views_test.py

```diff
@@ -1399,22 +1399,49 @@ def _test_mv_with_default_ttl(self, flush):
             assert_one(session, "SELECT * FROM mv", [1, 3, 1])
 
         # user provided ttl
-        self.update_view(session, "UPDATE t USING TTL 50 SET a = 4 WHERE k = 1", flush)
-        assert_one(session, "SELECT * FROM t", [1, 4, 1])
-        assert_one(session, "SELECT * FROM mv", [1, 4, 1])
+        start = time.time()
+        self.update_view(session, "UPDATE t USING TTL 100 SET a = 4 WHERE k = 1", flush)
+        try:
+            assert_one(session, "SELECT * FROM t", [1, 4, 1])
+            assert_one(session, "SELECT * FROM mv", [1, 4, 1])
+        except AssertionError as ae:
+            if (time.time() - start) > 100:
```

Comment:
Shouldn't all these inequalities be `>=`?

Reply:
They could. I can change that.